src/Ordering.Domain/AggregatesModel/BuyerAggregate/CardType.cs

```diff
@@ -1,20 +1,7 @@
-﻿using eShop.Ordering.Domain.SeedWork;
+﻿namespace eShop.Ordering.Domain.AggregatesModel.BuyerAggregate;
 
-namespace eShop.Ordering.Domain.AggregatesModel.BuyerAggregate;
-
-/// <remarks> 
-/// Card type class should be marked as abstract with protected constructor to encapsulate known enum types
-/// this is currently not possible as OrderingContextSeed uses this constructor to load cardTypes from csv file
-/// </remarks>
-public class CardType
-    : Enumeration
+public sealed class CardType
 {
-    public static CardType Amex = new(1, nameof(Amex));
-    public static CardType Visa = new(2, nameof(Visa));
-    public static CardType MasterCard = new(3, nameof(MasterCard));
-
-    public CardType(int id, string name)
-        : base(id, name)
-    {
-    }
+    public int Id { get; set; }
+    public string Name { get; set; } = string.Empty;
```

Comment:
I think it might be better to mark this as `required string` and provide an `init` for it?

Reply:
updated to `required` and `init`-only. also made the ID `init`-only, but let me know if there's an issue with doing that!